main.py

```diff
@@ -0,0 +1,90 @@
+# Task 1: Simple Quiz implementaion
+import json
+ 
+values = [4, 1, 1, 1, 3, 4, 1]
```

Comment:
This is not needed
please find another way to check for the result.
Hint: compare values